corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/content.less

```diff
@@ -73,6 +73,50 @@ body {
   height: 100%;
 }
 
+.marker-pin {
+  width: 30px;
+  height: 30px;
+  border-radius: 50% 50% 50% 0;
+  background: #c30b82;
+  position: absolute;
+  transform: rotate(-45deg);
+  left: 50%;
+  top: 50%;
+  margin: -15px 0 0 -15px;
+}
+// to draw white circle
+.marker-pin::after {
+    content: '';
+    width: 24px;
+    height: 24px;
+    margin: 3px 0 0 3px;
+    background: #fff;
+    position: absolute;
+    border-radius: 50%;
+ }
+
+.myDivIcon {
+  text-align: center;
+  /* Horizontally center the text (icon) */
+  line-height: 12px;
+  /* Vertically center the text (icon) */
+  color: #B4BCF5;
+  text-shadow: -1px 0 #212F78, 0 1px #212F78, 1px 0 #212F78, 0 -1px #212F78;
+}
+
+.list-cell-wrapper-style {
+  border: 1px solid white;
+  border-collapse: collapse;
+  vertical-align: top;
+  background-color: #f2f2f1;
+  container-type: inline-size;
+}
+
+.highlighted-case {
```

Comment:
Nitpick: I'd probably put this inside a `#module-case-list-container` to limit its scope. Same for `.list-cell-wrapper-style`. Might move the marker styles inside the map container div id, too. It's not that likely these class names will be reused elsewhere, but I think it's a good default to keep them constrained, and it's helpful when looking at styles to have a sense of where they're used.

Reply:
Oops, yes, it's an id not a class. I had just meant change the selector from `.highlighted-case` to `.module-case-list-container .highlighted-case`...but now that you mention it, yes, since this styling is only meant to be used with the case list, it ought to go into `module.less`.

Reply:
I realized the map styles can go in there too. How do I figure out which less file is appropriate?

Reply:
Spitballing here, but the two things I think about are:

Appropriate directory: most styles should go under `corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common`, and then for styles specific to either app preview of web apps we have 
 `corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp` and `corehq/apps/hqwebapp/static/preview_app`. Editorializing, I think the `preview_app` dir should be under cloudcare, a sibling of formplayer-common and formplayer-webapp. I'm also surprised we have so many files under `formplayer-webapp`: my mental model is that there are bunch of app preview styles to control the look of the the faux phone, but that not much styling would be used by web apps and **not** app preview.

Then for the file, it's just about whatever file is most specific. `content.less` is for styles that are very high level (`body`) or that don't fit anywhere else and don't justify making a new file (although it looks like we have some pretty small less files). Looking at it now, I don't think styles like `select-all-tile-checkbox` belong in `content.less`. I'm also not sure of the conceptual difference between `menu.less` and `module.less` - menu and module are synonyms in most of HQ, but this code might be using those terms differently.